.cargo/common.dic

```diff
@@ -0,0 +1,123 @@
+3
+-
+-
+(
+)
+*
+/
+@dev
+^
++
+<
+=
+>
+async
+backend
+backtraces
+Bech32
+behaviour
+benchmark
+benchmarking
+bip32
+blockchain
+BMT
+BoxFuture
+BoxStream
+breakpoint
+bytecode
+checksum
+CLI
+coinbase
+config/S
+cryptographically
+DA
+deduplicate
+deserialization
+deserialized
+dev
+Dev-note
+DHT
+discoverable
+DoS
+E2E
+endian
+enum
+env
+Ethereum
+FIXME
+Getters
+gossipsub
+GraphQL
+hardcoded
+HTTP
+HTTPS
+includable
+interoperability
+IP
+Jemalloc
+JSON
+Kademlia
+keygen
+keypair
+libp2p
+lifecycle
+MDNS
+merkle
+metadata
+MMR
+mplex
+mutex
+O_o
+opcode
+P2P
+parseable
+pinnable
+PoA
+Prometheus
+Pyroscope
+RAII
+Relayer/S
+resize
+retryable
+RFC
+RocksDB
+RPC
+runnable
+runtime
+serialization
+serialized
+sha256
+Sha256
+SMT
+sq_x
+startup
+stateful
+stateless
+stdin
+stdout
+struct
+substream
+sum_x_y
+TCP
+timestamp
+TODO
+tokio
+transactional
+TTL
+tx
+TxPool/S
+txs
+uncompress
+uri
+UTXO/S
```

Comment:
I prefer `UTxO` tbo, so maybe add that too or instead?

Reply:
Hmm. Maybe that's not standard:
https://en.wikipedia.org/wiki/Unspent_transaction_output

Althought I prefer it, maybe just stick with the standard.